.travis.yml

```diff
@@ -48,9 +47,9 @@ before_script:
 
 script:
   - set -ev
-  - mvn install -Drelease
+  - mvn clean install -Drelease
```

Comment:
Chilax, just checking if you had any experience with it on @travis-ci. No problem for me.